Pod/Classes/public/model/WistiaMedia.swift

```diff
@@ -95,7 +95,7 @@ public struct WistiaMedia {
      - Note: hlsMasterIndexManifestURL is valid for playback iff this returns true.
      **/
     public func hasHlsAssets() -> Bool {
-        return assets.contains { $0.type == "HlsVideoFile" }
+        return assets.contains { $0.type == "hls_video" }
```

Comment:
Since we see the type listed as both `hls_video` and `hlsVideoFile`, depending on which endpoint returns the asset :-/

And WistiaKit can playback media originating from either endpoint.

I suggest, as a more robust fix, checking for HLS assets  using `$0.type.lowercased().contains("hls")`